images/macos/tests/Common.Tests.ps1

```diff
@@ -369,4 +306,69 @@ Describe "Visual Studio For Mac" {
         $vsPath | Should -Exist
         $vstoolPath | Should -Exist
     }
+}
+
+Describe "AWS" {
+    It "AWS CLI" {
+        "aws --version" | Should -ReturnZeroExitCode
+    }
+    It "AWS SAM CLI" {
+        "sam --version" | Should -ReturnZeroExitCode
+    }
+
+    It "AWS Session Manager CLI" {
+        "session-manager-plugin --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Azcopy" {
+    It "Azcopy" {
+        "azcopy --version" | Should -ReturnZeroExitCode
+    }
+    It "Azure CLI" {

```

Comment:
Azure CLI should be in the separate describe. This tool is not related to AzCopy